challenges/objects-arrays.js

```diff
@@ -91,6 +140,11 @@ The zoos want to display both the scientific name and the animal name in front o
 
 */
 const displayNames = [];
+
+zooAnimals.forEach(function(animal){
+  return displayNames.push `${animal.animal_name} ${animal.scientific_name}`
```

Comment:
your return seems to have a syntax error --> this is what is returning [ [ '', ' ', '' ],
  'Jackal, asiatic',
  'Canis aureus',
  [ '', ' ', '' ],
  'Screamer, southern',
  'Chauna torquata',
  [ '', ' ', '' ],
  'White spoonbill',
  'Platalea leucordia',
  [ '', ' ', '' ],

- ^^ what is causing the ' ',' ',' '